doc/tutorial/text_analytics/solutions/exercise_02_sentiment.py

```diff
@@ -55,7 +55,11 @@
 
     # TASK: print the cross-validated scores for the each parameters set
     # explored by the grid search
-    print(grid_search.grid_scores_)
+    n_candidates = len(grid_search.candidate_params_)
+    test_split_keys = filter(lambda key: key.startswith('test_split_'),
+                             grid_search.results_.keys())
+    for i in range(n_candidates):
+        print(i, list(grid_search.results_[key][i] for key in test_split_keys))
```

Comment:
Is this showing each split score? Why do we want to print that out? Surely the previous version showed the result repr with mean, std, params, none of which you are showing here.